cli/jobs/pipelines/automl/cli-automl-classification-task-bankmarketing-pipeline/environment/preprocessing_env.yaml

```diff
@@ -0,0 +1,19 @@
+channels:
+  - conda-forge
+dependencies:
+  - python=3.8
+  - numpy=1.21.2
+  - pip=21.2.4
+  - scikit-learn=0.24.2
+  - scipy=1.7.1
+  - 'pandas>=1.1,<1.2'
+  - pip:
+      - 'inference-schema[numpy-support]==1.3.0'
+      - xlrd==2.0.1
+      - mlflow== 1.26.1
+      - azureml-mlflow==1.42.0
```

Comment:
azureml-mlfow pins the compatible mlflow version, any reason to have it separately?

Reply:
Done